templates/kubernetes/terraform/modules/kubernetes/metrics/prometheus/files/prometheus_operator_helm_config.yml

```diff
@@ -0,0 +1,2043 @@
+# Default values for prometheus-operator.
```

Comment:
The values in here are 99% the defaults for the prometheus-operator helm chart. I'm a little torn on whether to leave this file with all the options in it, or to exclude everything that is a default value..